### ISSUE TITLE
See all the metadata properties and how they are linked so that I can review if they are correct

### ISSUE THREAD

**Issue by ofanobilbao** [ebi-ait/dcp-ingest-central#6]

**Comment by aaclan-ebi** [comment]
We've ( @rdgoite @prabh-t @ESapenaVentura ) had a design discussion regarding this and it was identified that the main problem we want to solve here is identifying the orphaned entities as that implies that an entity doesn't belong to an experiment graph and will not be exported to TDR. 

We agreed that we want to prioritise first identifying these orphaned entities in a submission. That will be the only scope of this ticket. @ESapenaVentura is okay with it. (We might want to update the user story to reflect that e.g. As a wrangler... I want to view orphaned entities so that i can update their links...)  @clairerye @gabsie please confirm if you are okay with this scope?

The viewing of linked entities can be still be done in the (new) metadata details page which is functionality needed when updating the links. 

Thanks!

**Comment by clairerye** [comment]
Yes, I think that's the right way to split it!
So we will deliver this in two parts, with the identifying these orphaned entities in a submission first. And once that is done we will work on the more general summary?

**Comment by aaclan-ebi** [comment]
Hmm, is it okay if we put the general summary to a different user story ticket and put that in the Product Backlog for now?

Would that have a higher priority than the currently committed user stories for this sprint? We have to think more about how we want to implement it, the current way we model things wouldn't allow us to easily retrieve the summary.

I also remembered today, (when Laura brought up the KT on this) that Rolando implemented a feature to generate a summary for project. We can look into checking if that functionality is still working and that might be sufficient for our needs now.

**Comment by clairerye** [comment]
Ok, yes a separate ticket sounds good, I think it will therefore be the highest priority in the backlog but lower than the other 3 stories we have committed to. Are you happy to make the ticket?

**Comment by aaclan-ebi** [comment]
Yes, will do, thanks @clairerye !

**Comment by aaclan-ebi** [comment]
@clairerye I've moved the description of this user story to https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/42

And updated the description here. Please confirm if this looks good.

Thanks again!

**Comment by aaclan-ebi** [comment]
@rdgoite i added a ticket to update the UI for this user story, FYI.

**Comment by rays22** [comment]
When I use a [test file](https://drive.google.com/file/d/1ihv8P0HaJ-RPIZ-iFaInNexkyimq9nAD/view?usp=sharing)
with orphan biomaterial or data entities, then in column `linked` there is a yellow triangle with an exclamation mark. I mouse over I can see the message `no links with other entities`.
NOTE: However, orphaned entities can be identified from the submission page only if the entity is of type Biomaterals or Data. For example, orphaned entities in class Protocols appear to be allowed and pass the test. Is that the expected outcome?

**Comment by rays22** [comment]
Using [the test file as above](https://drive.google.com/file/d/1ihv8P0HaJ-RPIZ-iFaInNexkyimq9nAD/view?usp=sharing) I can confirm that I can identify all the orphaned entities in the tabs `Biomaterials`, `Protocols` and `Data`. I can not think a way to make a metadata spreadsheet that would create orphaned `Processes` when uploaded to Ingest UI, so I did not test that scenario.
* [TEST CASE ID: V1b](https://docs.google.com/spreadsheets/d/1sf_PnPRufEPUeqkADhewJGJsYsb00CBwymKKPFB_VHI/edit#gid=0) : **PASS**

**Comment by aaclan-ebi** [comment]
Thanks for testing this @rays22 !

Viewing of orphaned processes is still not implemented but I think we can already promote the changes tested so far to production on monday.

**Comment by clairerye** [comment]
Ability to filter by orphaned state to show all unlinked or linked entities

'filter by linking' dropdown similar to 'filter by state' as shown below with option to select to show 'linked' or to show 'unlinked' present in each of the tabs  

![image.png](https://images.zenhubusercontent.com/5e4d068a4ff78267228d27b4/a54df409-7ca3-4217-a3b4-25cde87b70d5)

**Comment by clairerye** [comment]
Can we also show what filters are set as we now are filtering by 2 aspects the potential for mixing up combinations of filters is much higher.

**Comment by mshadbolt** [comment]
i would be in favour of implementing filtering on each column rather than a big long list, something like this
![Screenshot 2020-12-02 at 09 29 25](https://user-images.githubusercontent.com/13007987/100854455-e9543400-3480-11eb-8c48-f575e2b524ea.png)
